FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -173,51 +188,66 @@ void TrajectorySeedProducer::produce(edm::Event& e, const edm::EventSetup& es)
 	return;
     }
     
-    // pointer to selected region
+    // pointers for selector function
     TrackingRegion * selectedTrackingRegion = 0;
+    auto pixelTripletGeneratorPtr = pixelTripletGenerator.get();
+    auto MultiHitGeneratorPtr = MultiHitGenerator.get();
```

Comment:
There is no need for this *Ptr objects.
Just use the .get() function on the unique_ptr whenever needed.